File: JULY-2025/01-07-2025/p3330.ts
function possibleStringCount(word: string): number {
    let preChar: string = word[0];
    let ans = 1;
    for (let i = 1; i < word.length; i ++) {
        if (word[i] !== preChar) preChar = word[i];
        else ans ++;
    }
    return ans;
};

File: JULY-2025/01-07-2025/p3330.swift
class Solution {
    func possibleStringCount(_ word: String) -> Int {
        let letters = Array(word)
        let n = letters.count
        var run = 1
        var count = 1
        for i in 1..<n {
            if letters[i] == letters[i - 1] {
                run += 1
            }
            else {
                if run > 1 {
                    count += run - 1
                }
                run = 1
            }
        }
        if run > 1 {
            count += run - 1
        }
        
        return count        
    }
}